recipe/meta.yaml

```diff
@@ -22,9 +22,9 @@ requirements:
     - make
     - vim
   host:
-    - kim-api
+    - kim-api >=2.2.1
   run:
-    - kim-api
+    - kim-api >=2.2.1
```

Comment:
```suggestion
    - {{ pin_run_as_build(kim-api, min_pin='x.x', max_pin='x.x') }}
```

Reply:
@ellio167 if the Openkim-models are compiled with one version of the kim-api is the version of the kim-api allowed to change or should it always be exactly the same?